### ISSUE TITLE
[Bug] [a11y] Consider making the "x" button from the ETP pop-up larger

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7718]
## Prerequisites
1. Make sure you have Google Accessibility Scanner installed.
2. Have a freshly installed build or a clean profile.

## Steps to reproduce
1. Launch Fenix and go to cnn.com.
2. Using the Accessibility Scanner scan the page with the ETP "Firefox Preview is blocking trackers on this site when the shield is purple." pop-up.

### Expected behavior
No suggestions are made.

### Actual behavior
The suggestion is to increase the item's size from 24dp x 24dp to 48dp x 48dp or larger - as seen in the screenshot below.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8), Samsung Galaxy Note 8 (Android 9)
* Fenix version: RC 3.0.2, Beta build 3.1.0-beta 3, Nightly build from 01/15

![for the 1st time_Accessibility Scanner](https://user-images.githubusercontent.com/40694158/72439340-6f0aba80-37af-11ea-827a-a443b8d81ae8.jpg)

**Comment by sblatz** [comment]
When this popup is displayed, it should say the text of the popup instead of just saying "popup window" 

The close button should say "close popup" 

The close button should be 48x48

**Comment by mcarare** [comment]
This should be already fixed. Can QA please confirm? TY!

**Comment by sflorean** [comment]
Verified as fixed on the latest Nightly build 6/3. Device: Samsung Galaxy Note 10 (Android 10).
